st2common/st2common/util/config_loader.py

```diff
@@ -99,21 +100,105 @@ def _get_values_for_config(self, config_schema_db, config_db):
         return config
 
     @staticmethod
-    def _get_object_property_schema(object_schema, additional_properties_keys=None):
+    def _get_object_properties_schema(object_schema, objecy_keys=None):
```

Comment:
Is this a typo, should that be object_keys?

Reply:
Ah yes. That's what I get for typing in my phone. I'll fix that.